tests/unit_test/data_processor/action_serializer_test.py

```
@@ -0,0 +1,601 @@
+from kairon.shared.actions.models import ActionType, ActionParameterType, DbActionOperationType
+from kairon.shared.callback.data_objects import CallbackConfig
+from kairon.shared.data.data_validation import DataValidation
+
+os.environ["system_file"] = "./tests/testing_data/system.yaml"
+Utility.load_environment()
+Utility.load_system_metadata()
+
+import pytest
+
+from kairon.shared.data.action_serializer import ActionSerializer
+from kairon.shared.data.data_objects import *
```

Ruff

`from kairon.shared.data.data_objects import *` used; unable to detect undefined names

(F403)